packages/cli/src/cmds/add.ts

```diff
@@ -70,12 +71,65 @@ export class AddCICD extends YargsCommand {
   }
 }
 
+export class AddSso extends YargsCommand {
+  public readonly commandHead = `sso`;
+  public readonly command = this.commandHead;
+  public readonly description = "Add Single Sign On for your project.";
+
+  public builder(yargs: Argv): Argv<any> {
+    this.params = HelpParamGenerator.getYargsParamForHelp("addSso");
+    return yargs.version(false).options(this.params);
+  }
+
+  public modifyArguments(args: { [argName: string]: any }): { [argName: string]: any } {
```

Comment:
seems this function does nothing?

Reply:
Followed code structure in addCICD above, not quite sure whether it is required.

Reply:
You can ask @Alive-Fish whether it required, if not required, you can remove it